frontend/public/module/k8s/get-resources.ts

```diff
@@ -91,7 +91,7 @@ const getResources_ = () => coFetchJSON('api/kubernetes/apis')
 
         const defineModels = (list: APIResourceList): K8sKind[] => list.resources.filter(({name}) => !name.includes('/'))
           .map(({name, singularName, namespaced, kind, verbs}) => {
-            const label = kind.replace(/([A-Z]+)/g, ' $1').slice(1);
+            const label = modelFor(kind) ? kind.replace(/([A-Z]+)/g, ' $1').slice(1) : kind;
```

Comment:
I don't think this is necessary. We should already use the value from the static model if present. I believe it could simply be

```js
   label: kind,
```

below and delete line 94.

Reply:
Maybe something like this:
```
const model = modelFor(kind);
const label = model ? model.label : kind;
```

Reply:
But it shouldn't be needed. We use the static model when defined, and only use the dynamic models when there's no static models. You shouldn't need to check `modelFor` at all.

Note that this check is no correct anyway. You also need to check API group. But again, it should be safe to remove.

Reply:
Ok, made the change and tested.